src/observable/FromEventObservable.ts

```diff
@@ -34,13 +35,28 @@ function isEventTarget(sourceObj: any): sourceObj is EventTarget {
 
 export type EventTargetLike = EventTarget | NodeStyleEventEmmitter | JQueryStyleEventEmitter | NodeList | HTMLCollection;
 
+export type EventListenerOptions = {
+  capture?: boolean;
+  passive?: boolean;
+  once?: boolean;
+} | boolean;
+
+export type SelectorMetodSignature<T> = (...args: Array<any>) => T;
+
 /**
  * We need this JSDoc comment for affecting ESDoc.
  * @extends {Ignored}
  * @hide true
  */
 export class FromEventObservable<T, R> extends Observable<T> {
 
+  /* tslint:disable:max-line-length */
+  static create<T>(target: EventTargetLike, eventName: string): Observable<T>;
+  static create<T>(target: EventTargetLike, eventName: string, selector: SelectorMetodSignature<T>): Observable<T>;
+  static create<T>(target: EventTargetLike, eventName: string, options: EventListenerOptions): Observable<T>;
+  static create<T>(target: EventTargetLike, eventName: string, selector: SelectorMetodSignature<T>, options: EventListenerOptions): Observable<T>;
```

Comment:
If that's the case, then what would happen if we swapped the selector and the options?
`(target, eventName, options, selector)`.